source/includes/api.html.md.erb

```diff
@@ -31130,7 +31202,7 @@ Puedes utilizar el siguiente reporte para exportar todos los datos de un mercado
           ad_type
         </strong>
         <small class="param__type">integer</small>
-        
+        <span class="param__required" title="Requerido">*</span>
```

Comment:
missing `required` symbol, this code made the change (already updated in wiki)

before
```
    return markRequiredParams(result);
```

after
```
    if (requiredParams.length) {
        return markRequiredParams(result);
    } else {
        return result;
    }
```